### DISCUSSION TITLE
Integration w/ existing react/node commenting systems (Commentami, Hypothesis, Annotator)

### DISCUSSION THREAD

**Discussion by juaneloDev** [outline/outline#3878] [Ideas]
I'm aware that development is ongoing for a full comments solution directly within Outline.

We have urgent needs for a commenting system and were considering integrating an existing open-source solution, either [Commentami](https://github.com/nearform/commentami), [Hypothesis](https://github.com/hypothesis/h), or [Annotator.js](https://annotatorjs.org/).

At the moment, Commentami seems like the most specific to our use-case (allows user tagging), as well as one having thorough documentation and docker-compose templates.

Is there anyone who has attempted to do this already that could point us in the right direction to get started?

**Comment by marijnvandevoorde** [comment]
Hypothesis can get you up & running with no extra effort. Well: create a group, add everyone to it and all install the chrome extension. It's not as great as it could be but it's a start.
